bakery/baker-client/src/test/scala/com/ing/bakery/client/BakerClientSpec.scala

```diff
@@ -97,6 +97,23 @@ class BakerClientSpec extends BakeryFunSpec {
       }
     }
 
+    test("scaladsl - balances") { context =>
+      val uri1 = Uri.unsafeFromString(s"https://localhost:${context.serverAddress.getPort}/")
+      val uri2 = Uri.unsafeFromString(s"https://invaliddomainname:445")
+      val uri3 = uri1 / "nowWorking"
+
+      BakerClient.resourceBalanced(
+        hosts = List(uri3, uri2, uri1),
+        pool = executionContext,
+        filters = List.empty,
+        tlsConfig = Some(clientTLSConfig))
+        .use { client =>
+          for {
+            response <- IO.fromFuture(IO(client.getAllRecipeInstancesMetadata))
+          } yield assert(response.isEmpty)
```

Comment:
what are we asserting exactly by `response.isEmpty`? 
how does it show that there were retries and one request was successful?